wordpress-mobile/WordPress-Aztec-iOS
# Improved customization of attachment visual properties

This PR allows better customisation of TextAttachment, and to make it at a global app level instead o need to do it for each attachment.

How to test:
 - Run the demo app
 - See if the customisation applied to TextAttachment on viewDidLoad are properly applied.

This will allow to fix [this kind of issue](https://github.com/wordpress-mobile/WordPress-iOS/issues/6706) directly on the app using the library.
